requirements/download-deps/portable-exe-lock.txt

```diff
@@ -202,9 +202,9 @@ pyinstaller==6.11.1 \
     --hash=sha256:ddc0fddd75f07f7e423da1f0822e389a42af011f9589e0269b87e0d89aa48c1f \
     --hash=sha256:e21c7806e34f40181e7606926a14579f848bfb1dc52cbca7eea66eccccbfe977
     # via -r requirements/portable-exe-extras.txt
-pyinstaller-hooks-contrib==2025.1 \
-    --hash=sha256:130818f9e9a0a7f2261f1fd66054966a3a50c99d000981c5d1db11d3ad0c6ab2 \
-    --hash=sha256:d3c799470cbc0bda60dcc8e6b4ab976777532b77621337f2037f558905e3a8e9
+pyinstaller-hooks-contrib==2025.2 \
```

Comment:
Meant to roll back this separate change that came in when regenerating the lockfiles.

Reply:
Fixed in f2a02f82568cfcbf1ae8514f5ef6c992ff9b9834